package.json

```diff
@@ -17,13 +17,12 @@
     "node": ">=8.0.0"
   },
   "dependencies": {
-    "hoek": "5.x.x",
-    "ioredis": "3.x.x"
+    "ioredis": "3 - 4"
   },
   "devDependencies": {
     "catbox": "10.x.x",
-    "code": "5.x.x",
-    "lab": "15.x.x"
+    "code": "^5.2.0",
```

Comment:
The hapi ecosystem defines dependencies in the format of `major.x.x`. Let’s keep this. Please revert the dependencies to match the format. Thank you!

Reply:
Never seen this, can you link me to where this is defined?
This is also dev dependency so does that even matter?